br/pkg/storage/local.go

```diff
@@ -77,7 +77,7 @@ func (l *LocalStorage) WalkDir(ctx context.Context, opt *WalkOption, fn func(str
 		// so use Rel to convert to relative path to l.base
 		path, _ = filepath.Rel(l.base, path)
```

Comment:
so this line can be removed

Reply:
I guess we'd better revert this change for keep the behavior consistent between `local` and `s3`.🤔